faker/providers/job/vi_VN/__init__.py

```diff
@@ -1,7 +1,6 @@
 # coding=utf-8
 from .. import Provider as JobProvider
 
-
 # https://vi.wikipedia.org/wiki/Thể_loại:Nghề_nghiệp
 
```

Comment:
@nkthanh98 there should be 2 empty lines before the class. 
```bash
faker/providers/job/vi_VN/__init__.py:6:1: E302 expected 2 blank lines, found 1
faker/providers/phone_number/vi_VN/__init__.py:7:1: E302 expected 2 blank lines, found 1
faker/providers/person/vi_VN/__init__.py:11:1: E302 expected 2 blank lines, found 1
faker/providers/color/vi_VN/__init__.py:9:1: E302 expected 2 blank lines, found 1
```
  
There was also too many empty lines after this line of code in __tests/providers/test_person.py__
```python
try:
    from unittest import mock
except ImportError:
    import mock
```